woocommerce/sensei-certificates
# Include dat files in package

Fixes #210

#### Changes proposed in this Pull Request:

* Includes the dat file in the final package.
* Also, the CSS files weren't minified. I've minified them.

#### Testing instructions:

* Package the plugin with `gulp package`.
* Verify dat file is included in `lib/tfpdf/font/unifont/dejavusanscondensed.cw.dat`
* Install package in a Sensei Install with completed students/certificates already.
* Add non-latin characters to certificate template.
* Verify certificate is rendered correctly.


#### Proposed changelog entry for your changes:
* Fix issue with non-Latin characters causing certificates to not render correctly.
